sspl_test/coverage/coverage_setup.py

```diff
@@ -0,0 +1,89 @@
+#! /usr/bin/python3
+import shutil
+import os
+import pwd
+import signal
+import psutil
+import time
+
+file_path = "/opt/seagate/cortx/sspl/low-level"
+report_path = "/var/cortx/sspl/coverage/sspl_xml_coverage_report.xml"
+
+def coverage_setup():
```

Comment:
Adding docstring can be useful for understanding this function for any new editors.

Reply:
is this okay?